manifests/params.pp

```diff
@@ -0,0 +1,15 @@
+class etherpad::params {
+  $default_padoptions = {
+    'nocolors'         => 'false',
```

Comment:
It might be better to change the type to actual booleans. That also means you can remove the puppet-lint warning. Added bonus is that you're guaranteed correct input from users.

Reply:
I understood what you are asking me to do. However, the Etherpad module allows you to use `String` for some options like` userColor`:

Example: userColor =% 23ff9900

But if you want, I put `Boolean` for all options except` lang`.

Reply:
Only if an option actually is a true Boolean, it should be a Boolean. You can also use `Variant[Boolean, Pattern[/[0-9a-fA-F]+/]]` or even `Variant[Boolean, String]`. It's just that your current code would produce invalid JSON for anything other than booleans now because those need quoting. That's why the `to_pretty_json` approach is so elegant: it deals with it for you.

Reply:
Fixed @ekohl.